addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -467,24 +467,24 @@ ColorpickerWidget.convertRgbToHsl = function (r, g, b) {
             saturation = delta / (1 - Math.abs(2 * lightness - 1));
         }
     }
-    hue = 60 * hue | 0;
+    hue = 60 * hue;
     return {
-        hue: hue < 0 ? hue += 360 : hue,
-        saturation: (saturation * 100) | 0,
-        lightness: (lightness * 100) | 0,
+        hue: hue < 0 ? hue + 360 : hue,
+        saturation: saturation * 100,
+        lightness: lightness * 100,
     };
 };
 /**
  * Converts HSL color components to RGB components.
  *
  * @static
- * @param {integer} h - [0, 360[
- * @param {integer} s - [0, 100]
- * @param {integer} l - [0, 100]
+ * @param {number} h - [0, 360[ (float)
+ * @param {number} s - [0, 100] (float)
+ * @param {number} l - [0, 100] (float)
```

Comment:
@oussamamessaoudi I made a proper commit out of this (with commit message and adapted the doc).